src/app/Login Screen Components/login/login.component.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Login and Registration Screen</title>
+  <link rel="stylesheet" href="login.component.css">
+</head>
+<body>
```

Comment:
Components don't have doctype, html, head, body tags. Components are individual, reusable parts of a page. You would only need the `div` level content of file.